feature-libs/cart/saved-cart/core/store/actions/saved-cart.action.ts

```diff
@@ -146,6 +158,13 @@ export class SaveCartFail extends StateUtils.EntityFailAction {
   }
 }
 
+export class RemoveSaveCartEntityProcess extends StateUtils.EntityRemoveAction {
```

Comment:
should extend EntityLoaderResetAction as mentioned previously to clear instead of Remove.